docs/sources/project/get-help.md

```diff
@@ -2,103 +2,127 @@ page_title: Where to chat or get help
 page_description: Describes Docker's communication channels
 page_keywords: IRC, Google group, Twitter, blog, Stackoverflow
 
+<style>
+/* @TODO add 'no-zebra' table-style to the docs-base stylesheet */
+/* Table without "zebra" striping */
+.content-body table.no-zebra tr {
+  background-color: transparent;
+}
+</style>
+
 # Where to chat or get help
 
-There are several communications channels you can use to chat with Docker community members and developers. 
+There are several communications channels you can use to chat with Docker
+community members and developers.
 
-<style type="text/css">
-.tg  {border-collapse:collapse;border-spacing:0;text-align: left;}
-.tg td{font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;vertical-align:top;}
-.tg th{font-family:Arial, sans-serif;font-size:14px;font-weight:normal;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;}
-</style>
-<table class="tg">
+<table>
   <col width="25%">
   <col width="75%">
   <tr>
-    <td class="tg-031e">Internet Relay Chat (IRC)</th>
-    <td class="tg-031e"><p>IRC a direct line to our most knowledgeable Docker users. The <code>#docker</code> and <code>#docker-dev</code> group  on <strong>irc.freenode.net</strong>. IRC was first created in 1988. So, it is a rich chat protocol but it can overwhelm new users. You can search <a href="https://botbot.me/freenode/docker/#" target="_blank">our chat archives</a>.</p>
-    Read our IRC quickstart guide below for an easy way to get started.</th>
+    <td>Internet Relay Chat (IRC)</th>
+    <td>
+      <p>
+        IRC a direct line to our most knowledgeable Docker users.
+        The <code>#docker</code> and <code>#docker-dev</code> group on 
+        <strong>irc.freenode.net</strong>. IRC was first created in 1988. 
+        So, it is a rich chat protocol but it can overwhelm new users. You can search
+        <a href="https://botbot.me/freenode/docker/#" target="_blank">our chat archives</a>.
+      </p>
+      Read our <a href="#irc-quickstart">IRC quickstart guide</a> below for 
```

Comment:
Can you remove the link you added? I design linking. In instructions, too many links are distracting and they redirect the reader's focus.   When I **do provide** links in the body of instructions, I specify a target attribute so out links open in a new tab rather than overwriting. You'll note all the outlinks open tabs.

In this case, I didn't provide the link to the instructions because they could scroll down --- and I want them too.